.gitignore

```diff
@@ -8,3 +8,7 @@
 /htdocs/search/mysite/conf/
 /htdocs/search/sphinx/
 /vendor/
+
+.idea/
```

Comment:
These kinds of ignores aren't really specific to the project and are more environment specific (ie: they depend on the dev's OS and IDE).

A better solution than having to decide which IDEs are worthy of being in a default ignore is to simply set up a global ignore on your machine so that all projects ignore them without modifying a project's own .gitignore.

This has 2 main advantages:
1. We don't pollute `.gitignore` files with non-project related rules and the associated maintenance that goes with this
2. Dev's don't have to amend every projects `.gitignore` to contain rules for common files on their specific environments

reference: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore